src/liboslexec/opmatrix.cpp

```diff
@@ -278,7 +288,12 @@ osl_transform_triple (void *sg_, void *Pin, int Pin_derivs,
             else
                 osl_transformn_vmv(Pout, &M, Pin);
         }
+#ifndef __CUDACC__
         else ASSERT(0);
```

Comment:
Is there merit to changing OIIO's ASSERT so that it's it's CUDA-safe? (Is that possible?)

Reply:
Sure, supposedly there is support for asserts in CUDA. I've never really used them though as it requires you to compile your CUDA code in debug mode. Thats probably just bad practice on my part though.